plaso/parsers/plist_plugins/ios_identityservices.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+"""Plist parser plugin for iOS com.apple.identityservices.idstatuscache.plist
+files."""
+
+from dfdatetime import cocoa_time as dfdatetime_cocoa_time
+
+from plaso.containers import events
+from plaso.parsers import plist
+from plaso.parsers.plist_plugins import interface
+
+
+class IOSIdstatusacheEventData(events.EventData):
+  """iOS com.apple.identityservices.idstatuscache.plist event.
+
+  Attributes:
+    process_name: (str) Name of the process that looked up an identifier.
+    apple_identifier: (str) type and value of the identifier.
+    lookup_time (dfdatetime.DateTimeValues): date and time of the lookup.
+  """
+
+  DATA_TYPE = 'ios:idstatuscache:lookup'
+
+  def __init__(self):
+    """Initializes event data."""
+    super(IOSIdstatusacheEventData, self).__init__(data_type=self.DATA_TYPE)
+    self.process_name = None
+    self.apple_identifier = None
+    self.lookup_time = None
+
+
+class IOSIdstatusachePlistPlugin(interface.PlistPlugin):
+  """Plist parser plugin for com.apple.identityservices.idstatuscache.plist
+  files."""
+
+  NAME = 'io_identityservices_idstatuscache'
+  DATA_FORMAT = 'Idstatuscache plist file'
+
+  PLIST_PATH_FILTERS = frozenset([interface.PlistPathFilter(
+      'com.apple.identityservices.idstatuscache.plist')])
+
+  def _GetDateTimeValueFromPlistKey(self, plist_key, plist_value_name):
+    """Retrieves a date and time value from a specific value in a plist key.
+
+    Args:
+      plist_key (object): plist key.
+      plist_value_name (str): name of the value in the plist key.
+
+    Returns:
+      dfdatetime.TimeElementsInMicroseconds: date and time or None if not
+          available.
+    """
+    timestamp = plist_key.get(plist_value_name, None)
```

Comment:
this will also return None if timestamp is 0 is that intentionally?

Reply:
I did not think of that case. My initial though was to avoid  returning an empty dfdatetime_cocoa_time.CocoaTime object.  

I could remove _GetDateTimeValueFromPlistKey entirely and simplify the code

Reply:
There are multiple case here timestamp is None or timestamp is 0

the first indicates there is no timestamp value, the second could potentially indicate the timestamp is not set.